tests/mock_network/churn.rs

```diff
@@ -723,8 +723,21 @@ impl Expectations {
         }
 
         for key in self.messages {
-            // All received messages for single nodes were removed: if any are left, they failed.
-            assert!(key.dst.is_multiple(), "Failed to receive request {:?}", key);
+            if let Authority::Node(curr_name) = key.dst {
+                let orig_name = new_to_old_map.get(&curr_name).unwrap_or(&curr_name);
+
+                if nodes
+                    .iter()
+                    .any(|node| node.inner.is_elder() && node.name() == *orig_name)
```

Comment:
This new code also seem to handle the case were a node was dropped.
How was it handled before, did elder destination not drop?

Reply:
I think dropping happens before message sending, so we never sent a message to dropped node.